taskflow/CMakeLists.txt

```diff
@@ -20,6 +20,7 @@ if (NOT Taskflow_FOUND OR INSTALL_TASKFLOW)
             CMAKE_CACHE_ARGS
             -DCMAKE_INSTALL_PREFIX:STRING=${CMAKE_INSTALL_PREFIX}
             -DCMAKE_BUILD_TYPE:STRING=Release
+            -DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON
```

Comment:
Does this have to be `-DCMAKE_EXPORT_NO_PACKAGE_REGISTRY:BOOL=ON`?

Reply:
Changed.